cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -108,7 +108,7 @@ func TestEvaluateExecute(t *testing.T) {
 			},
 
 			ExpectedOutputValidate: func(t *testing.T, output string, resultPath string) {
-				assert.Contains(t, output, `Evaluation score for "symflower/symbolic-execution" ("code-no-excess"): score=15, coverage-statement=10, files-executed=1, response-no-error=1, response-no-excess=1, response-not-empty=1, response-with-code=1`)
+				assert.Contains(t, output, `Evaluation score for "symflower/symbolic-execution" ("code-no-excess"): score=14, coverage-statement=10, files-executed=1, response-no-error=1, response-no-excess=1, response-with-code=1`)
```

Comment:
Please add a "why" to the commit where you remove the assessment not empty.

https://github.com/symflower/eval-dev-quality/pull/97/commits/4a0db2bfc6e0f6d892d6781e6c22205fd37e284c

i.e. because we don't need it anymore as empty responses are supposed to be errors